wpilibcExamples/src/main/cpp/examples/Frisbeebot/cpp/RobotContainer.cpp

```diff
@@ -47,5 +47,5 @@ void RobotContainer::ConfigureButtonBindings() {
 
 frc2::Command* RobotContainer::GetAutonomousCommand() {
   // Runs the chosen command in autonomous
-  return &m_autonomousCommand;
+  return *m_autonomousCommand;
```

Comment:
It's a bit unusual that dereferncing a CommandPtr gives you a pointer instead of a value type. Maybe `operator*()` could return a reference? If that was done, the user would have to write `&*m_autonomousCommand` like they would with std::unique_ptr.

std::unique_ptr also offers get() to return the underlying pointer, fyi.